internal/worker/accounts.go

```diff
@@ -24,9 +25,10 @@ var (
 )
 
 var (
-	minBalance  = types.Siacoins(1).Div64(2).Big()
-	maxBalance  = types.Siacoins(1)
-	maxNegDrift = new(big.Int).Neg(types.Siacoins(10).Big())
+	minBalance         = types.Siacoins(1).Div64(2).Big()
+	maxBalance         = types.Siacoins(1)
+	maxNegDrift        = new(big.Int).Neg(types.Siacoins(10).Big())
```

Comment:
I don't think I've seen a drift more than 1 SC. You might be able to get away with a lower value, like 4 SC - 5 SC.